libs/api/protocol/src/lib/core/command/helper/set-base-data-from-command-on.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { UserProducer } from '../../entity/partials/producer-partial.entity';
+import { ProtocolCommunicationEntryBase } from '../../entity/protocol-entries/protocol-entry-base.entity';
+import { BaseCreateMessageCommand } from '../base-create-message.command';
+
+export function setBaseDataFromCommandOnEntity(
```

Comment:
How about using the `mutate` functionality from automapper for this to avoid implementing simple mapping ourselves?

Reply:
this would mean we need a mapping for each command, i think this is fine for now, as the interface wont likely change